src/db.c

```diff
@@ -1742,8 +1784,16 @@ int expireIfNeeded(redisDb *db, robj *key, int flags) {
      * will have failed over and the new primary will send us the expire. */
     if (isPausedActionsWithUpdate(PAUSE_ACTION_EXPIRE)) return 1;
 
+    /* The key needs to be converted from static to heap before deleted */
+    int static_key = key->refcount == OBJ_STATIC_REFCOUNT;
+    if(static_key) {
```

Comment:
```suggestion
    if (static_key) {
```

Reply:
looks like i'm blind for white space issues recently :smile: